tests-unit/routesUser.js

```diff
@@ -42,37 +48,98 @@ describe('user', function () {
 		});
 	}
 
-	beforeEach(function (done) {
-		dao.deleteAllUsers(function (err) {
+	function configOptions(port, path, method, body, auth, version) {
```

Comment:
In Node (or JavaScript in general), if you have a function with lots of input parameters it may be considered a code-smell. Its really hard to read function calls like `configOptions('/test', 'GET', true, null, false, true)`. I'd rather have calls like this: 
```
configOptions({
 url: '/test',
 method: 'GET', 
 body: {/* some object */},
 auth: false, 
 version: true
});
```
Of course, this became obvious only after you made this refactor. And we have worst duplication problems. So, :+1: to you!